backend/src/main/java/com/engineering/thesis/backend/controller/AppointmentController.java

```diff
@@ -18,8 +20,13 @@ public void create(@RequestBody Appointment appointment) {
         appointmentService.create(appointment);
     }
 
+    @PutMapping("/update")
```

Comment:
We should have an extra EP to do the update, even though it works the same as create (added to most controllers).